### PR TITLE
Replace boilerplate with lombok

### PR DESCRIPTION
**Pull request by punkratz312**

### CODE REVIEW

**Comment by punkratz312** [comment]
Nice feature!

**Comment by benas** [comment]
Coool! Less is more :smile: 

Can you please resync with the master branch?

File: random-beans/pom.xml [hunk 1]
```diff
@@ -101,6 +101,12 @@
             <artifactId>junit-dataprovider</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.projectlombok</groupId>
+            <artifactId>lombok</artifactId>
+            <version>1.16.8</version>
+            <scope>provided</scope>
```
Comment (benas) [inline random-beans/pom.xml]: I think the scope here should be "test". Do you agree?

Comment (punkratz312) [inline random-beans/pom.xml]: i think it would not effect the target classed at all, but you´re right. we want 100% nothing less 👍

**Comment by benas** [comment]
Hey, why did you close this PR? That was a good idea to reduce boilerplate with lombok!

**Comment by punkratz312** [comment]
Sry, was a mistake - I´m not used to github. I will create a new one ^^

**Pull request closed without merging**